### PR TITLE
ZRZ-1012 include retweet links, use user UA, install UA-parser

### PR DESCRIPTION
**Pull request by ad-astra-via**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"root": true}
```
Comment (kuba-orlik) [inline .eslintrc.json]: we don't need this, having `root:true` in `.eslintrc.js` should be enough

Comment (ad-astra-via) [reply .eslintrc.json]: Removed it, thanks!

**Review by omarmosid — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -28,8 +28,9 @@
   },
   "homepage": "https://github.com/managed-components/twitter#readme",
   "devDependencies": {
-    "@managed-components/types": "^1.3.1",
+    "@managed-components/types": "^1.3.13",
     "@types/mustache": "^4.2.5",
+    "@types/ua-parser-js": "^0.7.39",
```
Comment (jonnyparris) [inline package.json]: is this still necessary without the ua-parser package?

File: src/index.ts [hunk 3]
```diff
@@ -89,60 +90,105 @@ export default async function (manager: Manager) {
...
-      manager.set('profileImage_' + user.screen_name, profileImage)
+      const res = await manager.fetch(user.profile_image_url_https, {
+        headers: {
+          Accept: 'image/jpeg,image/png,image/*,*/*;q=0.8',
+          'User-Agent': `${userAgent}`,
+        },
+        method: 'GET',
+      })
+      if (!res || !res.ok) {
+        throw new Error(
```
Comment (jonnyparris) [inline src/index.ts]: curious as to how big this blast radius is when it happens? do other Twitter MC actions still fire?

File: package.json (hunk 2 above)
Comment (ad-astra-via) [reply package.json]: Nope. Thanks!

File: src/index.ts (hunk 3 above)
Comment (ad-astra-via) [reply src/index.ts]: I removed it.

**Review by jonnyparris — APPROVED** [review]
🔥

**Pull request merged**